libraries/ms-helper/src/mcp2515.c

```diff
@@ -330,3 +333,384 @@ StatusCode mcp2515_tx(Mcp2515Storage *storage, uint32_t id, bool extended, uint6
 
   return STATUS_CODE_OK;
 }
+#else
```

Comment:
Same comment as gpio - it'd be nice to not have to duplicate all of this code, what if we want to update the x86 version? Then we have to change two spots!